### PR TITLE
Add feature to restart current test at results

### PR DESCRIPTION
**Pull request by MiConnell**
This PR adds functionality to press the `TAB` key once typing is complete to start again without the need to exit and rerun the `mitype` command.

### CODE REVIEW

**Comment by Mithil467** [comment]
Hey @MiConnell :wave:

Very glad to see your interest in contributing to mitype!
May you please create an feature request issue for this PR?

**Comment by MiConnell** [comment]
Hello! Feature request #33 has been created to address this.

File: mitype/app.py [hunk 1]
```diff
@@ -278,6 +286,12 @@ def update_state(self, win):
 
             win.addstr(" to see a replay! ")
 
+            win.addstr(self.line_count + 4, 0, " Press ")
+
+            win.addstr(" TAB ", curses.color_pair(5))
+
+            win.addstr(" to try a new test! ")
```
Comment (Mithil467) [inline mitype/app.py:293]: This message could be better writtten as "to retry".

**Review by Mithil467 — CHANGES_REQUESTED** [review]

File: mitype/app.py (hunk 1 above)
Comment (MiConnell) [reply mitype/app.py]: Updated the message.

**Review by Mithil467 — APPROVED** [review]
This looks good! Thank you for this fix @MiConnell! 😄

**Comment by MiConnell** [comment]
Glad I could contribute!

**Pull request merged**